### PR TITLE
Docs modification (admin)

### PR DESCRIPTION
**Pull request by Starmys**
Improve the language in admin manual

### CODE REVIEW

**Comment by suiguoxin** [comment]
> Improve the language in admin manual

Can you check also the `README.md` file in the root folder ?

File: docs/manual/cluster-admin/basic-management-operations.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 ## Management on Webportal
 
-The webportal provides some basic administration functions. If you log in to it as an administrator, you will find several buttons about administration on the left sidebar, as shown in the following image.
+The web portal provides some basic administration functions. If you log in to it as an administrator, you will find several buttons about the administration on the left sidebar, as shown in the following image.
```
Comment (suiguoxin) [inline docs/manual/cluster-admin/basic-management-operations.md:5]: `webportal` is a openpai service

File: docs/manual/cluster-admin/basic-management-operations.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ On the homepage, there is an `abnormal jobs` section for administrators. A job i
 
 ### Access Kubernetes Dashboard
 
-There is a shortcut to k8s dashboard on the webportal. However, it needs special authentication for security issues.
+There is a shortcut to the k8s dashboard on the web portal. However, it needs special authentication for security issues.
```
Comment (suiguoxin) [inline docs/manual/cluster-admin/basic-management-operations.md:39]: same here

File: docs/manual/cluster-admin/how-to-set-up-storage.md [hunk 3]
```diff
@@ -242,15 +242,15 @@ Do not omit any fields in `extension` or it will change the `virtualClusters` se
...
 
 Please read the document about [service management and paictl](./basic-management-operations.md#pai-service-management-and-paictl) first, and start a dev box container. Then, in the dev box container, pull the configuration by:
 
 ```bash
 ./paictl config pull -o /cluster-configuration
 ```
 
-To use storage manager, you should first decide a machine in PAI system to be the storage server. The machine **must** be one of PAI workers, not PAI master. Please open `/cluster-configuration/layout.yaml`, choose a worker machine, then add a `pai-storage: "true"` field to it. Here is an example of the edited `layout.yaml`:
+To use a storage manager, you should first decide on a machine in the PAI system to be the storage server. The machine **must** be one of PAI workers, not PAI master. Please open `/cluster-configuration/layout.yaml`, choose a worker machine, then add a `pai-storage: "true"` field to it. Here is an example of the edited `layout.yaml`:
```
Comment (suiguoxin) [inline docs/manual/cluster-admin/how-to-set-up-storage.md:253]: remove `a `

File: docs/manual/cluster-admin/how-to-add-and-remove-nodes.md [hunk 4]
```diff
@@ -119,7 +119,7 @@ machine-list:
     pai-worker: "true"
 ```
 
-- If you are using hived scheduler, you should modify its setting in `services-configuration.yaml` properly. Please refer to [how to set up virtual clusters](./how-to-set-up-virtual-clusters.md) and the [hived scheduler doc](https://github.com/microsoft/hivedscheduler/blob/master/doc/user-manual.md) for details. If you are using Kubernetes default scheduler, you can skip this step.
+- If you are using a hived scheduler, you should modify its setting in `services-configuration.yaml` properly. Please refer to [how to set up virtual clusters](./how-to-set-up-virtual-clusters.md) and the [hived scheduler doc](https://github.com/microsoft/hivedscheduler/blob/master/doc/user-manual.md) for details. If you are using Kubernetes default scheduler, you can skip this step.
```
Comment (suiguoxin) [inline docs/manual/cluster-admin/how-to-add-and-remove-nodes.md:122]: remove `a`

File: docs/manual/cluster-admin/how-to-set-up-virtual-clusters.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
...
+OpenPAI supports two kinds of schedulers: the Kubernetes scheduler, and [hivedscheduler](https://github.com/microsoft/hivedscheduler). [Hivedscheduler](https://github.com/microsoft/hivedscheduler) is a Kubernetes Scheduler for Deep Learning. It supports virtual cluster division, topology-aware resource guarantee, and optimized gang scheduling, which are not supported in the k8s default scheduler. If you didn't specify `enable_hived_scheduler: false` during installation, the hived scheduler is enabled by default. Please notice the only hived scheduler supports virtual cluster setup, k8s default scheduler doesn't support it.
```
Comment (suiguoxin) [inline docs/manual/cluster-admin/how-to-set-up-virtual-clusters.md:5]: Please remove the last `the`

File: docs/manual/cluster-admin/how-to-set-up-virtual-clusters.md [hunk 6]
```diff
@@ -251,7 +251,7 @@ hivedscheduler:
           cellNumber: 3
 ```
 
-In the above example, we set up 2 VCs: `default` and `v100`. The `default` VC has 2 K80 nodes, and `V100` VC has 3 V100 nodes. Every K80 node has 4 K80 GPUs and Every V100 nodes has 4 V100 GPUs.
+In the above example, we set up 2 VCs: `default` and `v100`. The `default` VC has 2 K80 nodes, and `V100` VC has 3 V100 nodes. Every K80 node has 4 K80 GPUs and Every V100 node has 4 V100 GPUs.
```
Comment (suiguoxin) [inline docs/manual/cluster-admin/how-to-set-up-virtual-clusters.md:255]: Every -> every

**Comment by Starmys** [comment]
Rebase operation error, will create a new one.

**Pull request closed without merging**